src/main/java/de/uni_tuebingen/ub/nppm/db/SaveHelper.java

```diff
@@ -47,6 +50,57 @@ public static List<Map<String, String>> selectAttribute(String table, String att
         return results;
     }
 
+    public static List<Map> selectAttributeMap(String table, String attributeString, int id) throws Exception {
```

Comment:
All function names that only use a SELECT statement should usually start with get... instead of select...

Reply:
done:  look at commit: improvement of select functions